napari/layers/utils/layer_utils.py

```diff
@@ -62,6 +62,26 @@ def _inner(func):
     return _inner
 
 
+def _nanmin(array):
+    """
+    call np.min but fall back to avoid nan and inf if necessary
+    """
+    min = array.min()
```

Comment:
```suggestion
    min = np.min(array)
```

`.min()` is not part of the [Array API](https://data-apis.org/array-api/latest/API_specification/array_object.html)

Reply:
This is pretty good info. Wonder if we should have an issue an scan the codebase for possible uses or misuses of the array API 🤔